### DISCUSSION TITLE
Google Chrome Omahaproxy Deprecated

### DISCUSSION THREAD

**Discussion by psomeren** [EUCPilots/evergreen-module#573] [Ideas]
A colleague introduced me to Evergreen for the Auto Update of Google Chrome. I've noticed Evergreen is still using the a .JSON-file from Omahaproxy. According to this page https://groups.google.com/a/chromium.org/g/chromium-dev/c/uH-nFrOLWtE, Omahaproxy should not be used any longer. Chromium Dash should be used instead.

I have build my own update mechanism using the URL https://versionhistory.googleapis.com/v1/chrome/platforms/**all**/channels/**all**/versions/**all**/releases, but takes to long and holds to much information. As you can see, you can replace "all" with specifics like this:

https://versionhistory.googleapis.com/v1/chrome/platforms/**win**/channels/**stable**/versions/all/releases

Which is much faster, as it only shows Platform Windows and Channel Stable.

**Comment by aaronparker** [comment]
The current version of Evergreen is using `https://versionhistory.googleapis.com/v1/chrome/platforms` to get Chrome version details. 

See #546
